packages/remix-eslint-config/internal.js

```diff
@@ -82,5 +82,13 @@ module.exports = {
         "prefer-let/prefer-let": OFF,
       },
     },
+    {
+      files: ["packages/**/*.*"],
+      excludedFiles: "**/__tests__/**/*.*",
+      rules: {
+        // Validate dependencies are listed in workspace package.json files
+        "import/no-extraneous-dependencies": "error",
```

Comment:
This will fail if a workspace file imports something that is not declared in the _workspace_ `package.json`.  This can go unnoticed currently if a workspace imports something that is a dependency of _another_ workspace and thus it exists in the root `node_modules`

Reply:
However - worth noting it will not fail in the PR status checks at the moment - we only run `lint --fix` on `main`/`dev` builds so it opens the potential to merge a PR that will break a dev build which is no bueno.  Should we run a subset (or all) lint rules on PRs?